components/eamxx/docs/user/model_input.md

```diff
@@ -1,8 +1,269 @@
-Model input
+Model inputs
 =====================================
 
-TODO: explain how defaults XML, atmchange/atmquery, buildml, and input.yaml work.
+This section explains how input parameters are passed to EAMxx, and how the user can
+change their value. The full list of the currently configuraable runtime parameters for
+EAMxx can be found [here](../common/eamxx_params.md).
 
-[Here](../common/eamxx_params.md) is a list of the currently configurable runtime parameters for EAMxx.
+The main files/scripts involved in the EAMxx runtime configuration are the following:
 
+1. `buildnml`: this script, located in `components/eamxx/cime_config`, is called by CIME's case management
+    scripts (`case.setup`, `case.build, `case.submit`), and is responsible for creating EAMxx input files,
+    including handling of user-specific modifications for the runtime parameters. Users should not have to
+    modify this script, nor should they have to manually call it.
+2. `scream_input.yaml`: EAMxx will parse this [YAML](https://yaml.org/spec/1.2.2) file at runtime,
+    to detect all of its configuration parameters. It is generated automatically by `buildnml` inside the
+    RUNDIR/data folder, where `RUNDIR` is the run directory generated by CIME. Since this file is automatically
+    generated when `buildnml` runs, users should not manually modify it. Any manual modification will be
+    lost the next time `buildnml` runs (e.g., at `case.submit` time).
+3. 'namelist_scream.xml`: this XML file is generated by `buildnml`, and contains all the runtime parameters that EAMxx
+    will read in at runtime. `buildnml` uses this XML file as an intermediate file during the generation of
+    `scream_input.yaml`. In particular, `buildnml` generates this file using case information to select the
+    proper configuration from the file `namelist_defaults_scream.xml`, located in `components/eamxx/cime_config`.
+    As it was the case for `scream_input.yaml`, since this XML file is automatically generated by `buildnml`,
```

Comment:
"As it was the case for `scream_input.yaml`,": better grammar would be "As with `scream_input.yaml`,"